Sofa/Component/MechanicalLoad/src/sofa/component/mechanicalload/PlaneForceField.inl

```diff
@@ -33,6 +33,36 @@
 #include <limits>
 #include <sofa/simulation/Node.h>
 
+namespace
+{
+    template<typename DataTypes>
+    void get3DFrameFromDPosNormal(const typename DataTypes::DPos& dposnormal, sofa::type::Vec3& v1, sofa::type::Vec3& v2, sofa::type::Vec3& normal)
+    {
+        for (auto i = 0; i < dposnormal.size() && i < 3; i++)
+            normal[i] = dposnormal[i];
+
+        // find a first vector inside the plane
+        if (std::abs(normal[0]) > 0.0)
+        {
+            v1 = sofa::type::Vec3(-normal[1] / normal[0], 1.0, 0.0);
+        }
+        else if (std::abs(normal[1]) > 0.0)
+        {
+            v1 = sofa::type::Vec3(1.0, -normal[0] / normal[1], 0.0);
+        }
+        else if (std::abs(normal[2]) > 0.0)
+        {
+            v1 = sofa::type::Vec3(1.0, 0.0, -normal[0] / normal[2]);
+        }
+
+        v1.normalize();
```

Comment:
`v1.normalize();` returns a bool. That would be great to check it. Especially because the previous conditions do not check that `normal` is 0.

Reply:
Ackchyually 🤫, I was thinking this component does not check (at all) if the normal given in the Data<> to create the plane forcefield is correct, and one would need to add a callback which check the data and set the component to valid/invalid if normal is wrong.
(and I just wanted to fix the warning 😭)

Reply:
so @fredroy?  just the warning or the callback? 😆